src/Microsoft.SqlTools.ServiceLayer/Profiler/IProfilerSessionMonitor.cs

```diff
@@ -27,6 +27,6 @@ public interface IProfilerSessionMonitor
         /// <summary>
         /// Stops monitoring a profiler session
         /// </summary>
-        bool StopMonitoringSession(string sessionId);
+        bool StopMonitoringSession(string sessionId, out ProfilerSession session);
```

Comment:
Does any code actually use the `bool` return value? If not then it might be better to just have it return the session.